app/base/handlers.js

```diff
@@ -12,9 +12,14 @@ exports.home = (req, res, next) => {
   const rstudio_is_deploying = req.session.rstudio_is_deploying;
   req.session.rstudio_is_deploying = false;
 
-  Tool.list()
-    .then((tools) => {
-      res.render('base/home.html', { tools, get_tool_url, rstudio_is_deploying });
+  Promise.all([Tool.list(), User.get(req.user.auth0_id)])
+    .then(([tools, user]) => {
```

Comment:
Might be good to update `req.user` here with `user`

Reply:
Then you won't have to pass `user` to the template because `current_user` will be up to date, and then you don't need the template changes.

Reply:
I don't follow... would updating `req.user` change `app.locals.current_user`? I tried simply assigning it to `user` and nothing changed...

Reply:
Leaving as is, as discussed. 👍